charts/zqd/templates/deployment.yaml

```diff
@@ -52,7 +62,21 @@ spec:
               path: /status
               port: http
           env:
-          ##- if eq .Values.serviceAccount.name "" ##
+          {{- if eq .Values.personality "worker"}}
+          - name: ZQD_REGISTER_ADDR
```

Comment:
How about delete this, and just rely on `ZQD_RECRUITER_ADDR` as configured below?